notebooks/hybrid_rag_pipeline_with_breakpoints.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
```

Comment:
<p>There's an extra "\n" on the first line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>